### DISCUSSION TITLE
[CDK validium] the confirmation status does not change

### DISCUSSION THREAD

**Discussion by JacksonRGB** [blockscout/blockscout#9667] [Q&A]
Hello, I have a question about Blockscout and Validium. I am running a test network for Validium, but when I run Blockscout (production-zkevm branch), the confirmation status in transaction details always remains 'confirmed by sequencer' and does not change to 'l1 confirmed'. I would like to know if there is a need to set any environment variables for this.

![image](https://github.com/blockscout/blockscout/assets/39642866/f0584fb9-7f0e-4a2b-97e1-3e05622ef9f5)

**Comment by GimluCom** [comment]
Have you configured the [zkEVM ENVs](https://docs.blockscout.com/for-developers/information-and-settings/env-variables#polygon-zkevm-rollup-management)?

**Reply by JacksonRGB** [reply]
@GimluCom thank you!

fixed. upgrade cdk validium version
